server/test/repository/AccountRepositoryTest.java

```diff
@@ -418,13 +418,13 @@ public void updateIdTokens() {
     Instant timeInPast = now.minus(1, ChronoUnit.SECONDS).toInstant(ZoneOffset.UTC);
     JWT expiredJwt = getJwtWithExpirationTime(timeInPast);
 
-    repo.updateIdTokens(account, "sessionId1", expiredJwt.serialize());
+    repo.addIdTokenAndPrune(account, "sessionId1", expiredJwt.serialize());
```

Comment:
Blocking- Can you please update the test method name?

Reply:
Good catch thanks!